src-tauri/tauri.conf.json

```diff
@@ -62,8 +62,13 @@
       {
         "fullscreen": false,
         "resizable": true,
-        "title": "FlightCore"
+        "decorations": false,
+        "title": "FlightCore",
+        "height": 600,
+        "minHeight": 600,
+        "width": 1000,
+        "minWidth": 1000
```

Comment:
We probably have enough space in UI to make the min size a bit smaller (mostly useful when developing FlightCore on a laptop where screen real estate is precious)

The smallest screen it should work in is SteamDeck which is 1280x800px, so current resolution is fine for that ^^

Reply:
What do you think minimal dimensions should be?

Reply:
```suggestion
        "height": 600,
        "minHeight": 300,
        "width": 1000,
        "minWidth": 600
```

600x300 seems to be the smallest we can handle atm ^^
Note that the default size is unchanged, only the minsize the window can be resized to.

![screenshot](https://user-images.githubusercontent.com/40122905/193682458-2a2c5987-dc32-471d-a078-2a837edf5fd2.png)